tests/test_plugin.py

```diff
@@ -365,3 +364,19 @@ def test_skip_missing_correlation_tests(api_client, pytest_test_items):
 
     assert not pytest_test_items[0].get_closest_marker('skip')
     assert not pytest_test_items[1].get_closest_marker('skip')
+
+
+def test_check_timeout_convert_string():
```

Comment:
This test is not actually testing the plugin, you need to used the `api_client` fixture.

Reply:
Dear @allankp ,
Thanks for your comment.
I've just modified the unit test.
Please have a look.
Peace!!!